### DISCUSSION TITLE
Is There any Support Planned for Ubuntu 22.04?

### DISCUSSION THREAD

**Discussion by barmanroys** [autogluon/autogluon#2377] [Ideas]
Tried installing autogluon on an AWS EC2 instance running Ubuntu 22.04, but it failed. From some online reading, I have been led to believe the Ubuntu 22.04, Python 3.10 stack is not supported yet, although I am not so sure which is the culprit here between Ubuntu or Python. Am I correct in assuming it cannot be done, or is there any configuration or other necessary package that I need to install to make autogluon work with this? 

Also, can I ask if supporting the latest version of Ubuntu and Python is something on the roadmap?

**Comment by gradientsky** [comment]
We have not worked yet to try and check both ubuntu 22.04 and python 3.10, so this is unexplored area yet. If you do have any notes on this topic, feel free to start issue thread - similar to [M1](https://github.com/awslabs/autogluon/issues/1242)/[M2](https://github.com/awslabs/autogluon/issues/2271) discussions.
